modules/users/tests/server/user.server.model.tests.js

```diff
@@ -345,36 +345,37 @@ describe('User Model Unit Tests:', function () {
       
     });
 
-    it('should not allow double dotted characters in email address - "abcdef@abc..com"', function (done) {
```

Comment:
@lirantal Starting with this test, the `else` block is checking that `should.not.exist(err)` at line 360. This assert will never pass if it makes it to this line, since `err` existing is how the code will enter this `else` block.